samples/gocdk-runtimevar/main_test.go

```diff
@@ -37,61 +35,11 @@ func Test(t *testing.T) {
 		err error
 	}
 
-	var (
-		ctx    context.Context = context.Background()
-		cancel func()
-		outc   chan result
-	)
+	ctx := context.Background()
 
 	runtimevar := cmdtest.InProcessProgram("gocdk-runtimevar", func() int {
 		return run(ctx)
 	})
 	ts.Commands["gocdk-runtimevar"] = runtimevar
-
```

Comment:
Why all this?

Reply:
See the PR description: I had to drop some stuff from the gocdk-runtimevar sample; it was racy and got 100% racy with go 1.14.